main.go

```diff
@@ -45,6 +45,7 @@ type Config struct {
 	ZipName                       string `env:"zip_name"`
 	DeployPath                    string `env:"deploy_path"`
 	NotifyUserGroups              string `env:"notify_user_groups"`
+	AlwaysNotifyUserGroups        string `env:"always_notify_user_groups"`
```

Comment:
Do we need to validate the contents of this on the step side? Just asking because we explicity state what are the allowed values in the input description.

Reply:
That's good call, the website is permissive, but I will add a similar validation to the step: https://github.com/bitrise-io/bitrise-website/pull/15590/files#diff-73cf7fcaa573ba233dfb2f0cbbaf69854dcc9a004fca260a11ca1c4a4ed9f2a6R382-R397